rskj-core/src/test/java/co/rsk/config/BridgeConstantsTest.java

```diff
@@ -122,4 +123,22 @@ void test_getMinimumPeginTxValue(BridgeConstants bridgeConstants, boolean isRSKI
             assertEquals(bridgeConstants.legacyMinimumPeginTxValue, minimumPeginTxValue);
         }
     }
+
+    private static Stream<Arguments> bridgeConstantsArgProvider() {
+        return Stream.of(
+            Arguments.of(BridgeMainNetConstants.getInstance()),
+            Arguments.of(BridgeTestNetConstants.getInstance()),
+            Arguments.of(BridgeRegTestConstants.getInstance())
+        );
+    }
+
+    @ParameterizedTest()
+    @MethodSource("bridgeConstantsArgProvider")
+    void test_getBtcHeightWhenPegoutTxIndexActivates(BridgeConstants bridgeConstants){
+        // Act
+        long btcHeightWhenPegoutTxIndexActivates = bridgeConstants.getBtcHeightWhenPegoutTxIndexActivates();
+
+        // assert
+        Assertions.assertTrue(btcHeightWhenPegoutTxIndexActivates > 0);
```

Comment:
Shouldn't you be checking the expected value for each network? This assertion is a bit weak, and we risk forgetting about changing it. Even if you end up changing the value for each network, we'd still have the test done and it will fail after the values are changed so that will also work as a remainder that the test needs to be updated.

Reply:
Cool. I assigned random values to each network and asserted against those random values. Once we set the final real value, the test will fail as expected.